src/modules/compliance/src/lib/procedures/ensureFilePermissions.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "audit": {
+      "anyOf": [
+        {
+          "type": "object",
+          "required": [
+            "ensureFilePermissions"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "ensureFilePermissions": {
+              "type": "object",
+              "required": [
+                "filename",
+                "user",
+                "group",
+                "permissions"
```

Comment:
Since CIS provides masks - could you make `mask` mandatory and `permissions` an optional field?
Here and in remediate.

Reply:
Actually only filename is mandatory (since we might only want to check user/group), fixed.